src/components/editor/TranscriptEditor.tsx

```
@@ -247,43 +249,118 @@ export function TranscriptEditor({
+                      word.deleted
+                        ? "text-muted-foreground/30 line-through scale-95"
+                        : "text-foreground hover:bg-muted",
+                      isActive &&
+                        "bg-primary text-primary-foreground shadow-sm scale-110 z-10",
+                    )}
+                    title={
+                      word.deleted
+                        ? "Right-click to restore"
+                        : "Click to seek, Right-click to delete"
+                    }
+                  >
```

Biome (2.4.15)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)